.env

```diff
@@ -0,0 +1,5 @@
+PYTHON_VERSION=3.5
```

Comment:
I suggest using `env.template` file instead of this so that users can modify it locally after doing

```sh
cp env.template .env
```

This is common practice which helps to have predefined `.env` file but changing it won't affect git tree state and allows users to integrate it to their workflow better because there are many tools that use `.env` file and many users have it in their project root already